### ISSUE TITLE
Read-byte might return :eof when used with a gray stream -> error in drakma

### ISSUE THREAD

**Issue by kpoeck** [clasp-developers/clasp#1026]
```lisp
CL-USER> (drakma:http-request "http://files.rcsb.org/view/1BLJ.pdb")
;;; Warning: Problems determining charset (falling back to binary):
:EOF is not of type INTEGER.
````

**Comment by kpoeck** [comment]
Testing a fix

**Comment by kpoeck** [comment]
fix merged (#1027 )
